UnityProject/Assets/Scripts/Items/Science/Clothing/ReactiveArmor.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections.Generic;
+using Gateway;
+using Systems.Teleport;
+using UnityEngine;
+
+namespace Items.Science.Clothing
+{
+	/// <summary>
+	/// A special type of armor that reacts to being hit. Randomly teleporting the wearer to a different location when getting hit.
+	/// </summary>
+	public class ReactiveArmor : MonoBehaviour, IServerInventoryMove
+	{
+		private Pickupable pickupable;
+
+		[SerializeField] private List<DamageType> blacklistedDamageTypes = new List<DamageType> {
+			DamageType.Clone,
+			DamageType.Stamina,
```

Comment:
wouldn't it mean it wouldn't trigger on disablers, though we don't have implemented yet?

Reply:
When it's properly implemented, we can worry about that.

Reply:
cool